src/components/SafeListSidebar/SafeList/SafeListItem.tsx

```diff
@@ -95,17 +117,19 @@ const SafeListItem = ({
     <ListItem button onClick={handleOpenSafe} ref={safeRef}>
       <StyledIcon type="check" size="md" color="primary" checked={isCurrentSafe} />
       <StyledPrefixedEthHashInfo hash={address} name={safeName} shortName={shortName} showAvatar shortenHash={4} />
-      <ListItemSecondaryAction>
+      <StyledListItemSecondaryAction>
         {ethBalance ? (
-          `${formatAmount(ethBalance)} ${nativeCurrencySymbol}`
+          <StyledText size="lg">
+            {formatAmount(ethBalance)} {nativeCurrencySymbol}
+          </StyledText>
         ) : showAddSafeLink ? (
-          <Link to={generateSafeRoute(LOAD_SPECIFIC_SAFE_ROUTE, routesSlug)} onClick={handleLoadSafe}>
-            <Text size="sm" color="primary">
+          <StyledButton onClick={handleLoadSafe} size="md" variant="outlined">
+            <Text size="lg" color="primary">
```

Comment:
Is there a particular reason you increased the `size`?

Reply:
To make it the same size as the safe balance text size in the List for consistency. I don't mind keeping it at a smaller size like before.